contracts/FlowCallbackScheduler.cdc

```diff
@@ -15,12 +15,28 @@ access(all) contract FlowCallbackScheduler {
         timestamp: UFix64?,
         priority: UInt8,
         executionEffort: UInt64,
-        fees: UFix64
+        fees: UFix64,
+        callbackOwner: Address
     )
 
-    access(all) event CallbackProcessed(id: UInt64, executionEffort: UInt64)
-    access(all) event CallbackExecuted(id: UInt64)
-    access(all) event CallbackCanceled(id: UInt64)
+    access(all) event CallbackProcessed(
+        id: UInt64,
+        priority: UInt8,
```

Comment:
this is not needed for FVM

Reply:
I want to keep it anyway in case there are third parties that want to follow events and need more info, like @bjartek was asking for